.github/workflows/flow-pull-request-formatting.yaml

```diff
@@ -49,6 +49,14 @@ jobs:
         uses: step-security/conventional-pr-title-action@0eae74515f5a79f8773fa04142dd746df76666ac # v1.0.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          daemon-config: |
+            {
+              "registry-mirrors": [
+                "https://artifacts.swirldslabs.io/central-docker-external/"
+              ]
+            }
+
```

Comment:
We shouldn't add this to this action. The daemon-config is for docker engine setup.